### PR TITLE
v0.15 Release Notes

### PR DESCRIPTION
**Pull request by denniszollo**
Release branch notes

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Piksi Firmware Release Notes
 
 Contents
 --------
-
+ * [v0.15](#v0.14)
```
Comment (gsmcmullin) [inline RELEASE-NOTES.md]: Should link to v0.15

**Comment by gsmcmullin** [comment]
Otherwise, it looks good.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -11,6 +11,45 @@ Contents
...
+   * Fix "LD Factorization" message  & bug
+   * Fix for suddden large baseline bug when dropping & reacquiring sats
+ * Send RTK absolute positions as outputs over the MSG_POS_LLH (id 0x0201) and
+   MSG_POS_ECEF (id 0x0200) SBP messages - (Please review the SBP documentation
+   of the flags field for both messages)
+    * A flag of 0 represents single point position
+    * A flag of 1 represents the float solution
+    * A flag of 2 represents the fixed RTK solution
+ * Add log level prefixes to messages prints (ERROR, WARN, INFO & DEBUG)
+ * Remove SBP functionality from libswiftnav and use libsbp instead to handle
```
Comment (mookerji) [inline RELEASE-NOTES.md]: Link? http://github.com/swift-nav/libsbp

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -11,6 +11,39 @@ Contents
...
+#### Console / serial_link.py
+
+ * Add display of psuedo absolute positions on solution tab
+ * Annotate settings to provide a description and notes for each setting
+ * Add Json serialization option to serial link functionality
+ * Provide a progress indicator for firmware updates
+ * Add a pause and clear button to the console log
+ * Add Piksi serial number to the console window header
+ * Use libsbp to decode messages from Piksi
+ * If running console from the source, update dependencies or rerun the setup.sh
```
Comment (mookerji) [inline RELEASE-NOTES.md]: Watchdog timer option on Piksi heartbeat messages.

**Pull request merged**